packages/server/src/notification/notification.service.ts

```diff
@@ -88,16 +95,70 @@ export class NotificationService {
   }
 
   // notifies a user via phone number
-  async notifyPhone(pn: PhoneNotifModel, message: string): Promise<void> {
-    try {
-      this.twilioClient &&
-        (await this.twilioClient.messages.create({
-          body: message,
-          from: this.configService.get('TWILIOPHONENUMBER'),
-          to: pn.phoneNumber,
-        }));
-    } catch (error) {
-      console.error('problem sending message', error);
+  async notifyPhone(
+    pn: PhoneNotifModel,
+    message: string,
+    verificationMethod: boolean,
+  ): Promise<void> {
+    if (verificationMethod || pn.verified) {
+      try {
+        this.twilioClient &&
+          (await this.twilioClient.messages.create({
+            body: message,
+            from: this.configService.get('TWILIOPHONENUMBER'),
+            to: pn.phoneNumber,
+          }));
+      } catch (error) {
+        console.error('problem sending message', error);
+      }
+    }
+  }
+
+  async verifyPhone(
+    phoneNumber: string,
+    message: string,
+    response: Response,
```

Comment:
Services should not deal with the http layer. I think you could have this function return a string and have the controller send that string back and set the content type